test/testcafe/spec/DeviceChallengePollViewOktaVerify_spec.js

```diff
@@ -22,9 +22,8 @@ async function setup(t) {
 test(`should have the correct content`, async t => {
   const deviceChallengePollPageObject = await setup(t);
   await t.expect(deviceChallengePollPageObject.getHeader()).eql('Verify account access');
-  await t.expect(deviceChallengePollPageObject.getSubtitle()).eql('Launching Okta Verify...');
   await t.expect(deviceChallengePollPageObject.getContent())
-    .eql('If nothing prompts from the browser, click here to launch Okta Verify, or make sure Okta Verify is installed.');
+    .eql('Launching Okta Verify...\n\nIf nothing prompts from the browser, click here to launch Okta Verify, or make sure Okta Verify is installed.');
```

Comment:
Can you break the header and content into two separate checks? It seems to be easy to break if we rely on the two line breaks.